src/libraries/skinDynLib/test/skinContactTest.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia (IIT)
+ * CopyPolicy: Released under the terms of the GNU GPL v2.0.
+ */
+
+#include <cassert>
+#include <cmath>
+#include <ctime>
```

Comment:
regarding the old discussion with @diegoferigo on `ctime`  vs `chrono`
The `chrono` code is a bit trickier.
```cpp
auto start = std::chrono::system_clock::now();

/* do some work */

auto end = std::chrono::system_clock::now();
auto elapsed = end - start;
std::cout << elapsed.count() << '\n';
```
(and I would try with `high_resolution_clock` instead of `system_clock`)